src/main/scala/config/MetricsSettings.scala

```diff
@@ -3,25 +3,39 @@ package config
 import com.typesafe.config.Config
 import scala.collection.JavaConverters._
 
-case class RegistryOptions(enabled: Boolean = true, intervalSeconds : Int)
-case class CloudWatch(enabled: Boolean = false, name : String, intervalMinutes: Int)
-case class DataDog(enabled: Boolean = false, name : String, intervalMinutes: Int, agentHost: String, agentPort: Int, trackedConsumerGroups: List[String])
+case class RegistryOptions(enabled: Boolean = true, intervalSeconds: Int)
+case class CloudWatch(enabled: Boolean = false, name: String, intervalMinutes: Int)
+
+case class DataDog(enabled: Boolean = false,
+                   name: String,
+                   intervalMinutes: Int,
+                   agentHost: String,
+                   agentPort: Int,
+                   trackedConsumerGroups: List[String],
+                   remove_tags_from_metric_name: Boolean)
```

Comment:
This change wasn't necessary, `case class` param names should remain "camel-cased"😉

Reply:
replaceAll FTW :/